client/my-sites/hosting/sftp-card/index.js

```diff
@@ -196,7 +196,7 @@ export const SftpCard = ( {
 						) }
 					</p>
 					<Button onClick={ createUser } primary className="sftp-card__create-credentials-button">
-						{ translate( 'Create SFTP Credentials' ) }
+						{ translate( 'Create SFTP credentials' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.